### PR TITLE
No Issues found

### PR DESCRIPTION
**Pull request by luisrpavanelli**
In this PR, we worked, me and @Mylo16,  with check accessibility, no issues are found!

List
Page Titles - No issues found
Image text alternatives ** (alt text) - No issues found
Text headers - No issues found
Color Contrast - No issues found
Resize - No issues found
Interaction Labels ( are optional but recommended as voiceover tools can read them and must be set to invisible to match figma design ) - No issues found
Moving content - No issues found
Multimedia - No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -284,7 +284,7 @@ <h2 id="contactformtext">I'm always interested in hearing about new projects, so
             </ul>
         </div>
         <div class="container">
-            <p id="contactemail">luisrpavanello@gmail.com</p>
+            <p id="contactemail">luisrpavanello@gmail.com</p>
```
Comment (iambenkis) [inline index.html:287]: - [ ] Keep doing the great work :+1:

**Review by iambenkis — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey Team :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Basic Structure is good
:heavy_check_mark: Can be maximized upto 200%


Good job on implementing some of the requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Required changes :recycle: 

- [ ] Instead of using `h2` here :blush: Please delete it and kindly use only  `a` because this it does not respect  the heading of your page and according to the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :+1:
https://github.com/luisrpavanelli/Portfolio-Project/blob/8cd95faa2eff26e926d1ca71aacc0270b314b2f3/index.html#L21-L23

- [ ] The `see project`  does not have enough contrast of 2.6 Please add enough contrast for this button , the color of when the hovering , focusing on the element should be contrasted, it should be at least 4:5:1 instead of 1.

Screenshot from 2023-03-01 01-12-46

Screenshot from 2023-03-01 01-13-17

- [ ] You have made a lot of effort here, but please add a label for each input item for the accessibility purpose :+1: 
https://github.com/luisrpavanelli/Portfolio-Project/blob/8cd95faa2eff26e926d1ca71aacc0270b314b2f3/index.html#L257-L262


# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @iambenkis in your question, so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
In this PR, we worked, me and @Mylo16, with check accessibility, no issues are found!

List
Page Titles - No issues found
Image text alternatives ** (alt text) - No issues found
**Text headers - Modified
Color Contrast - Modified**
Resize - No issues found
**Interaction Labels ( are optional but recommended as voiceover tools can read them and must be set to invisible to match figma design ) - Add labels**
Moving content - No issues found
Multimedia - No issues found

**Required changes:
- We change to respect the heading of my page and according to the heading hierarchy)
- We change all the shadow bottons like suggestion above
- We made and add labels in contact forms to get more accessibility**

File: index.html [hunk 2]
```diff
@@ -18,9 +18,9 @@
         <nav>
             <img id="menuicon" src="./Desktop_Icons/menuhb.png" atl="Menu" />
             <ul class="menutabs">
-                <a href="#"><li><h2 id="textform2">Portfolio</h2></li></a>
-                <a href="#"><li><h2 id="textform2">About</h2></li></a>
-                <a href="#"><li><h2 id="textform2">Contact</h2></li></a>
+                <uma href="#"><li><a id="textform2">Portfolio</a></li></uma>
+                <uma href="#"><li><a id="textform2">About</a></li></uma>
+                <aum href="#"><li><a id="textform2">Contact</a></li></uma>
```
Comment (DuaneDave) [inline index.html:23]: As previously requested, these items should be wrapped in an `a` tag just as the previous reviewer said but I can see that you have made a mistake in doing so, hence I recommend you remove this `<uma>` tag which is not an HTML tag as it is making the supposed links not clickable and use an `a` tag here in order to improve the accessibility and to aid users that are dependent on a screenreader

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @luisrpavanelli and @Mylo16 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
✔️ Good PR summary
✔️ Proper ranking of header on the page


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
In this PR, we worked, me and @Mylo16, with check accessibility, no issues are found!

List
Page Titles - No issues found
Image text alternatives ** (alt text) - No issues found
Text headers - Modified
Color Contrast - Modified
Resize - No issues found
Interaction Labels ( are optional but recommended as voiceover tools can read them and must be set to invisible to match figma design ) - Add labels
Moving content - No issues found
Multimedia - No issues found

**Required Changes:

We change to respect the heading of my page and according to the heading hierarchy)
We change all the shadow bottons like suggestion above
We made and add labels in contact forms to get more accessibility**

**Required Changes 2: 

We change to respect the heading of my page and according to the heading hierarchy)

**Review by souad988 — APPROVED** [review]
Hi @luisrpavanelli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6a4a5143327075565a70544d4f34765573302f67697068792e676966



## Highlights
✔Use correct github workflow
✔Good PR summary

## Optional suggestions
_Kindly use a PR title that better describes the work done in this feature

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**